pdf_parser/pdf_parse.py

```diff
@@ -110,7 +110,7 @@ def parse_pdf_document_pdftxt(document):
     """
 
     parsed_path = document.name.replace('.pdf', '.html')
-    os.system('pdftotext -q -bbox %s %s 1>/dev/null 2>/dev/null' % (
+    os.system('pdftotext -q -bbox \'%s\' \'%s\' 1>/dev/null 2>/dev/null' % (
```

Comment:
does `.format` not work? isn't this the recommended way to string formating?

Reply:
It is. Will change that 👍